services/business/checkers/route_rule_checker.go

```diff
@@ -42,15 +45,16 @@ func (in RouteRuleChecker) runGroupChecks() models.IstioValidations {
 }
 
 // enabledCheckersFor returns the list of all individual enabled checkers.
-func enabledCheckersFor(object kubernetes.IstioObject) []Checker {
+func (in RouteRuleChecker) enabledCheckersFor(object kubernetes.IstioObject) []Checker {
```

Comment:
I am not sure if I like to extract functions for one-line assign.
Indeed I am adding this lines without the enabledCheckersFor, sorry, I guess a comment is enough in assignment :-).